32 t18 create order page 2/2

irtassedat opened the pull request:
tamm anlamıyla içime sinmedi biraz daha görsel iyileştirmesi yapmam gerekecek fakat önceki turuncul halinden çook daha iyi gözüktüğü kesin

irtassedat committed:
"Defined reducers and actions for payment information"

irtassedat committed:
"Revised primary and secondary buttons for enhanced clarity and simplicity."

gokhanozdemir commented on src/components/LayoutComponents/Header.jsx:
LocalStorage işlemleri her render'da çalışmak yerine, örneğin useEffect ile kontrol edilmeli veya custom bir hook içinde yönetilmelidir. Bu, performansı ve kodun yeniden kullanılabilirliğini artırır.

gokhanozdemir replied:
isLoggedIn state'i ve handleLogout fonksiyonunda kullanıcı oturumunu kontrol etmek için doğrudan localStorage kullanılıyor. Bu işlemler, daha güvenli bir yöntem olan Redux store üzerinden yönetilmelidir.

gokhanozdemir commented on src/store/actions/shoppingCartAction.jsx:
```diff
@@ -1,6 +1,11 @@
 import { TOGGLE_PRODUCT_CHECKED } from '../reducers/shoppingCartReducer';
 import axiosInstance from '../../mock/axiosInstance';
-
+import {
```
Bunlar actionsdan import edilmeli

irtassedat committed:
Start Improvement: Initial Enhancements for Create Order Page

irtassedat committed:
feat: Update address/payment info display and edit button styles

- Align address info within the active tab
- Add edit button next to address info

irtassedat committed:
feat: Update UI for PaymentOptions and CreateOrderPage

- Improved layout and styling for address and payment sections
- Added new form validation for address input
- Enhanced user interaction with dynamic tab switching
- Implemented order submission functionality
- Fixed issues with state management and UI updates

irtassedat committed:
refactor: Update CreateOrderPage with order submission handling and Redux actions

- Create action types for CREATE_ORDER_SUCCESS, CREATE_ORDER_FAILURE, and RESET_CART
- Update handleOrderSubmit to properly dispatch createOrder action and handle responses
- Add createOrderSuccess, createOrderFailure, and resetCart actions in shoppingCartActions
- Implement case handling for new actions in shoppingCartReducer
- Ensure cart reset and error handling after order submission

irtassedat committed:
Fetch product details on order submission, add product descriptions to order data, and handle product fetching errors.

irtassedat committed:
Create and enhance PreviousOrderPage: Implement initial layout, fetch orders, and display order details with modal and blur background effect.

irtassedat committed:
refactor: Update product card layout and navigation
- Improve UI for product cards with better layout and design
- Add navigation to product details page from the product card
- Refactor hover effects and interaction elements

irtassedat committed:
refactor: Update product card layout and navigation
- Improve UI for product cards with better layout and design
- Add navigation to product details page from the product card
- Refactor hover effects and interaction elements

irtassedat committed:
"refactor: Enhance create order page with better payment options
- Improve layout and handling of payment options
- Update order summary display for better user experience
- Ensure proper handling of order submission"

irtassedat committed:
"
- Enhance UI for payment options section
- Improve user interaction and experience"

irtassedat committed:
git commit -m "refactor: Improve product details page with carousel and better UI
- Integrate carousel for product images
- Enhance user interaction elements on the product details page
- Improve layout and design for better usability"

irtassedat committed:
git commit -m "refactor: Revamp shopping cart page layout and functionality
- Update cart item layout with better design and spacing
- Fix issue with cart item removal"

irtassedat committed:
git commit -m "feat:
- Ensure proper state updates and error handling for cart actions"

irtassedat commented:
@gokhanozdemir 
vercel failleri gerçekten çok fazla can sıkıyor farkındayım düzelteceğim fakat iş başvuruları dolasıyıla şuanda mergeliyorum.
